### PR TITLE
Fix 100+ broken links

### PR DESCRIPTION
**Pull request by pbakaus**
This PR fixes an ton amount of broken links across all sections:

* Many were actually broken and didn't redirect
* Others redirected, but redirected to the wrong pages

All fixes were applied manually and tested locally without rewrite rules to ensure they lead to the correct resource.

Note: There are still around ~40 bad links (from originally ~200 or so) that should be addressed in another PR.

### CODE REVIEW

**Comment by petele** [comment]
@pbakaus - do you remember which ones 404'ed, the redirects **should** have caught that, which means we missed a few of those and I'd like to fix that too.

File: src/content/en/fundamentals/getting-started/your-first-multi-screen-site/responsive.markdown [hunk 1]
```diff
@@ -201,7 +201,7 @@ where we want to change it.
 
 600px appears to be a good place to create our first breakpoint as it will give us scope
 to reposition elements to make them fit the screen better.  We can do this
-using a technology called [Media Queries]({{site.fundamentals}}/layouts/rwd-fundamentals/#use-css-media-queries-for-responsiveness).
+using a technology called [Media Queries]({{site.fundamentals}}/design-and-ui/responsive/fundamentals/use-media-queries).
```
Comment (petele) [inline src/content/en/fundamentals/getting-started/your-first-multi-screen-site/responsive.markdown]: can you remove `{{site.fundamentals}}` on this one and just go with `/web/fundamentals/` please

File: src/content/en/showcase/case-study/service-workers-iowa.markdown [hunk 2]
```diff
@@ -564,12 +564,12 @@ This ended up causing an issue for us during development, when we started using
 traffic was coming from. For example, we [added](https://github.com/GoogleChrome/ioweb2015/blob/28113917b88436dd569c39fd5eef184b6aefdd1c/app/scripts/shed/push-notifications.js#L32)
 the `utm_source=notification` parameter to URLs that were opened when clicking on one of our
 notifications, and used `utm_source=web_app_manifest` in the [`start_url`](https://github.com/GoogleChrome/ioweb2015/blob/0bab714dbb08927f901420fc05b43b9f97f7ddc3/app/templates/manifest.json#L4)
-for our [web app manifest](https://developers.google.com/web/updates/2014/11/Support-for-installable-web-apps-with-webapp-manifest-in-chrome-38-for-Android?hl=en).
+for our [web app manifest](/web/updates/2014/11/Support-for-installable-web-apps-with-webapp-manifest-in-chrome-38-for-Android?hl=en).
```
Comment (petele) [inline src/content/en/showcase/case-study/service-workers-iowa.markdown]: we can prob drop the `?hl=en` here

File: src/content/en/shows/cds/2013/blink-behind-the-scenes.markdown [hunk 3]
```diff
@@ -21,7 +21,7 @@ One big change we made when we forked from Blink was to add an intents system: e
 
 Then, on our [public mailing list](https://groups.google.com/a/chromium.org/forum/#!topic/blink-dev/) we announce an [intent to ship](https://groups.google.com/a/chromium.org/forum/#!topic/blink-dev/yujPcy889e4).
 
-At [chromestatus.com](http://chromestatus.com) you can see the [features](www.chromestatus.com/features) we've worked on, the features we've shipped, and those we're planning to deprecate. You can also check the [Chromium Releases blog](http://googlechromereleases.blogspot.co.uk/), which has links to bugs and to our tracker dashboard.
+At [chromestatus.com](http://chromestatus.com) you can see the [features](http://www.chromestatus.com/features) we've worked on, the features we've shipped, and those we're planning to deprecate. You can also check the [Chromium Releases blog](http://googlechromereleases.blogspot.co.uk/), which has links to bugs and to our tracker dashboard.
```
Comment (petele) [inline src/content/en/shows/cds/2013/blink-behind-the-scenes.markdown]: can we change it to `https`?

File: src/jekyll/_includes/page-structure/close-page.liquid [hunk 4]
```diff
@@ -40,9 +40,9 @@
 
       <ul class="wf-footer__list wf-footer__list--secondary">
         <li><a href="http://www.google.com/">Google</a></li>
-        <li><a href="http://developers.google.com/site-terms">{{"terms_of_service" | localize_string}}</a></li>
-        <li><a href="http://www.google.com/intl/en/privacy/">{{"privacy_policy" | localize_string}}</a></li>
-        <li><a href="http://developers.google.com/careers">{{"careers" | localize_string}}</a></li>
+        <li><a href="https://developers.google.com/site-terms">{{"terms_of_service" | localize_string}}</a></li>
+        <li><a href="http://www.google.com/intl/en/policies/privacy">{{"privacy_policy" | localize_string}}</a></li>
```
Comment (petele) [inline src/jekyll/_includes/page-structure/close-page.liquid]: can we make this one `https` too please?

File: src/content/en/fundamentals/getting-started/your-first-multi-screen-site/responsive.markdown (hunk 1 above)
Comment (gauntface) [inline src/content/en/fundamentals/getting-started/your-first-multi-screen-site/responsive.markdown]: {{site.WFBaseUrl}}/fundamentals/

**Comment by gauntface** [comment]
Just wanted to say this is epic work @pbakaus - thanks a lot.

**Comment by jpmedley** [comment]
🙌

Joe Medley | Technical Writer, DevPlat | jmedley@google.com | 816-678-7195

**Comment by petele** [comment]
I'm going to do a push tomorrow morning (ET), if you can get these updates in by then, I'll include them, thanks!!

**Comment by pbakaus** [comment]
Just saw the follow-ups, will address and attempt to fix a few more! I purposely used a link crawler that crawls my local instance, and since that doesn't have redirect mappings, it reported even working redirects as 404. In those cases, I manually looked at the live site and used the correct redirect link.

I would say about half were actually bad links (or bad redirects), and the other half worked, but it's still nice to fix them.

**Comment by gauntface** [comment]
@pbakaus what crawler were you using? Looking to build a tool to just
manage all this stuff.

**Comment by pbakaus** [comment]
@gauntface the crawler is called Integrity, not sure if it has a command line interface. Worked really well for my purposes. Would definitely be nice to automate it all :)

**Comment by jpmedley** [comment]
On Thu, Oct 22, 2015 at 12:04 PM, Paul Bakaus <notifications@github.com>
wrote:

> Would definitely be nice to automate it all :)


+100

Joe Medley | Technical Writer, DevPlat | jmedley@google.com | 816-678-7195

File: src/content/en/fundamentals/getting-started/your-first-multi-screen-site/responsive.markdown (hunk 1 above)
Comment (pbakaus) [inline src/content/en/fundamentals/getting-started/your-first-multi-screen-site/responsive.markdown]: Fixed! I used Pete's notation, as the /web/... syntax i used throughout the site.

File: src/content/en/showcase/case-study/service-workers-iowa.markdown (hunk 2 above)
Comment (pbakaus) [inline src/content/en/showcase/case-study/service-workers-iowa.markdown]: Removed hl=en from all links across the site.

File: src/content/en/shows/cds/2013/blink-behind-the-scenes.markdown (hunk 3 above)
Comment (pbakaus) [inline src/content/en/shows/cds/2013/blink-behind-the-scenes.markdown]: changed to https across all links on the site, wherever possible.

File: src/jekyll/_includes/page-structure/close-page.liquid (hunk 4 above)
Comment (pbakaus) [inline src/jekyll/_includes/page-structure/close-page.liquid]: yep! same as above, now done for the entire page (fingers crossed).

**Comment by pbakaus** [comment]
Fixed even more bad links and addressed all comments. PR should be in pretty good shape, PTAL.

**Pull request merged**